stock_quant_merge/models/stock.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# © 2015 OdooMRP team
+# © 2015 AvanzOSC
+# © 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class StockQuant(models.Model):
+    _inherit = 'stock.quant'
+
+    @api.multi
+    def merge_stock_quants(self):
+        pending_quants = self.filtered(lambda x: True)
```

Comment:
Yes, I did, but I don't remember why :worried: 
Maybe you should also start by doing the for only with those quants that do not have reservation...
I mean `for quant2merge in self.filtered(lambda x: not x.reservation_id)`